### PR TITLE
[device/celestica-blackstone]: Update switch CPLD driver follow FPGA i2c buses

### PR DESCRIPTION
**Pull request by mudsut4ke**
#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106

### CODE REVIEW

File: platform/broadcom/sonic-platform-modules-cel/blackstone/modules/misc_cpld.c [hunk 1]
```diff
@@ -17,27 +15,24 @@
 #include <linux/module.h>
 #include <linux/slab.h>
 #include <linux/i2c.h>
-#include <linux/mutex.h>
 #include <linux/hwmon.h>
 
 #define CPLD1_ADDR      0x30
-#define CPLD2_ADDR      0x31
+// #define CPLD2_ADDR      0x31
```
Comment (pphuchar) [inline platform/broadcom/sonic-platform-modules-cel/blackstone/modules/misc_cpld.c:21]: Can this be uncommented or removed?

File: platform/broadcom/sonic-platform-modules-cel/blackstone/modules/misc_cpld.c [hunk 2]
```diff
@@ -280,149 +253,88 @@ static struct attribute_group sff_led_groups = {
...
-                err = -EINVAL;
-                goto exit;
-        }
+        struct misc_cpld_data *drvdata1;
+        struct device *hwmon_dev1;
 
         if (!i2c_check_functionality(client->adapter, I2C_FUNC_I2C))
                 return -EPFNOSUPPORT;
 
         /* CPLD1 */
```
Comment (pphuchar) [inline platform/broadcom/sonic-platform-modules-cel/blackstone/modules/misc_cpld.c:266]: nitpick: consider remove this comment.

**Review by pphuchar — COMMENTED** [review]
As commented.

**Comment by mudsut4ke** [comment]
> As commented.

updated in : https://github.com/mudsut4ke/sonic-buildimage/pull/1779/commits/656c763373b1c05634f8278cc2d1c6b601b0f262

**Pull request merged**